File: week_6/pset_6/sentimental-mario-less/mario.py
from cs50 import get_int

# Prompt from the user
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
# Build pyramid
for i in range(1, height + 1):
    for j in range(1, height + 1):
        if i + j <= height:
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: week_1/pset_1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt the pyramid height until its in range
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //Create the pyramid
    for (int i = 1; i <= height; i++) //Iterate the rows
    {
        for (int j = 1; j <= height; j++) //Iterate the columns
        {
            if (i + j <= height)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}